integration-cli/daemon.go

```diff
@@ -61,7 +63,11 @@ func NewDaemon(c *check.C) *Daemon {
 	err := os.MkdirAll(daemonSockRoot, 0700)
 	c.Assert(err, checker.IsNil, check.Commentf("could not create daemon socket root"))
 
-	id := fmt.Sprintf("d%d", time.Now().UnixNano()%100000000)
+	randomBytes := make([]byte, 8)
+	_, err = rand.Read(randomBytes)
+	c.Assert(err, checker.IsNil, check.Commentf("could not read bytes in daemon id generation"))
+	id := fmt.Sprintf("d%d", binary.BigEndian.Uint64(randomBytes)%100000000)
```

Comment:
Didn't know about stringId, but that looks like it should do exactly what I want